scripts/badger/registry.py

```diff
@@ -2,20 +2,99 @@
 from great_ape_safe import GreatApeSafe
 from helpers.addresses import get_registry as registry
 
-console = Console()
+C = Console()
 
 # assuming techOps
-contract_registry = registry()
-chain_safe = GreatApeSafe(contract_registry.badger_wallets.techops_multisig)
-chain_safe.init_badger()
+r = registry()
```

Comment:
i think this behaviour was simplified by just importing `r` from `helpers.addresses` no?